cosmos/genesis.go

```diff
@@ -111,7 +111,8 @@ func genGenesisAppState(defaultGenesisŚtate map[string]json.RawMessage, signedS
 	genAccs := []genaccounts.GenesisAccount{}
 	for _, signer := range signedStdTx.GetSigners() {
 		stakes := sdktypes.NewCoin(sdktypes.DefaultBondDenom, sdktypes.NewInt(100000000))
-		genAcc := genaccounts.NewGenesisAccountRaw(signer, sdktypes.NewCoins(stakes), sdktypes.NewCoins(), 0, 0, "", "")
+		mesg := sdktypes.NewCoin("mesg", sdktypes.NewInt(1000000000000000))
```

Comment:
This should be done better. But maybe it's useless as PR https://github.com/mesg-foundation/engine/pull/1552 could be use to generate the genesis